xarray/core/formatting_html.py

```diff
@@ -183,7 +183,8 @@ def array_section(obj):
     # "unique" id to expand/collapse the section
     data_id = "section-" + str(uuid.uuid4())
     collapsed = ""
-    preview = escape(inline_variable_array_repr(obj.variable, max_width=70))
+    variable = obj.variable if hasattr(obj, "variable") else obj
+    preview = escape(inline_variable_array_repr(variable, max_width=70))
```

Comment:
```suggestion
    preview = escape(inline_variable_array_repr(getattr(obj, "variable", obj), max_width=70))
```
you should probably run `black` after that (or assign the result of `getattr` to `variable`)

Reply:
I believe I have black formatting on save (but I've been doing PRs on other repos and
black just reformats everything -- because they don't use it -- so I have to
turn it off)

Reply:
OK `whats-new.rst` updated

Reply:
that's fine, it's just that I'm not sure if my suggestion would need to be reformatted. So what I meant was that if you commit via github's interface, you might have to run `black`.

Reply:
TBC the `black` issue is not really that important, I wanted to propose replacing the conditional assignment
```python
variable = ... if hasattr(obj, ...) else ...
```
with a `getattr` call with a default:
```python
variable = getattr(obj, ..., obj)
```

Reply:
yep that's better